src/routes/views/app/RegisterView.tsx

```diff
@@ -0,0 +1,11 @@
+import RegisterForm from '../../../components/base/RegisterForm'
+
+const RegisterView = () => {
```

Comment:
You should probably add the state you have in the `RegisterForm` here.

Reply:
added useState to RegisterForm